clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -22872,6 +22872,10 @@ llvm::CallInst *CodeGenFunction::EmitFPBuiltinIndirectCall(
     // only if it has an fpbuiltin intrinsic.
     unsigned BuiltinID = getCurrentBuiltinID();
     Name = CGM.getContext().BuiltinInfo.getName(BuiltinID);
+    if (!FD->getNameInfo().getName().isIdentifier() ||
```

Comment:
As I mentioned in Teams, this doesn't look like the right fix to me. I think the problem is that we somehow got here when the call wasn't an indirect call. I think it would be better to check that in EmitCall before EmitFPBuiltinIndirectCall is called.